.github/workflows/colcon.yaml

```diff
@@ -0,0 +1,49 @@
+name: Colcon Build Test
+on:
+  push:
+    # branches:
+    # - 'rolling'
+    # - 'iron'
+    # - 'humble'
```

Comment:
@Ryanf55 Why do you comment out branches?

Reply:
Ah, unintentional just to make this easier to iterate on without having a PR. I've dropped that commit.

Now, CI only runs this job if
* It's a PR
* It's a commit on any of those three branches (which happens just after a merge).